File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Lab/src/P08_RefactorVolumeOfPyramid.java
import java.util.Scanner;
public class P08_RefactorVolumeOfPyramid {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        double length, width, height = 0.0;

        System.out.print("Length: ");
            length = Double.parseDouble(console.nextLine());
        System.out.print("Width: ");
            width = Double.parseDouble(console.nextLine());
        System.out.print("Height: ");
            height = Double.parseDouble(console.nextLine());

        double volume = (length * width * height) / 3;
        System.out.format("Pyramid Volume: %.2f%n", volume);
    }
}

File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Lab/P08_RefactorVolumeOfPyramid/P08_RefactorVolumeOfPyramid.cs
using System;

namespace P08_RefactorVolumeOfPyramid
{
    class P08_RefactorVolumeOfPyramid
    {
        static void Main(string[] args)
        {
            double length, width, height = 0.0;

            Console.Write("Length: ");
            length = double.Parse(Console.ReadLine());
            Console.Write("Width: ");
            width = double.Parse(Console.ReadLine());
            Console.Write("Height: ");
            height = double.Parse(Console.ReadLine());

            double volume = (length * width * height) / 3;
            Console.WriteLine($"Pyramid Volume: {volume:F2}");
        }
    }
}
